e2e-webdriver/src/app.e2e.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { AppPage } from './page-objects/app.po';
+import { ProductDetailPage } from './page-objects/product-detail.po';
+
+describe('workspace-project App', async () => {
+    let page: AppPage;
+    let detailPage: ProductDetailPage;
+
+    beforeEach(() => {
+        page = new AppPage();
+        detailPage = new ProductDetailPage();
+    });
+
+    it('should display title', async () => {
+        await page.navigateTo();
+        expect(await page.getTitle()).toEqual('GET YOUR ICE CREAM');
+    });
+
+    it('should navigate to correct detail page when product was clicked', async () => {
+        await page.navigateTo();
+        const productName = 'Solero';
+        const $product = await page.getProductLinkByName(productName);
```

Comment:
Ist der Präfix mit `$` ein Merkmal für einen Selektor? Ich finde das ungewöhnlich und würde das `$` weglassen.

Reply:
Ja, ich habe das in Anlehnung an JQuery gemacht, wo das üblich ist. Ist in diesem Fall nur Gewohnheit, kann ich rausnehmen.

Reply:
Mit jQuery wollen wir nichts zu tun haben ;-)

Reply:
Das will niemand 😏